docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -314,9 +314,21 @@ scenario, and input draw.
   * - Person-time in each Alzheimer's cause state
     -
     -
-  * - Test counts (CSF, PET, BBBM)
-    - By diagnosis provided (for BBBM tests)
+  * - Test counts
+    - * Test type (CSF, PET, BBBM). 
+      * Diagnosis provided (for BBBM tests). 
+      * Disease state at time of test (BBBM, MCI, AD). 
+      * Age at time of test. 
+      * CSF/PET status (has simulant received CSF/PET test before). 
+      * *Positive* BBBM test status (has simulant recieved positive BBBM test before).
+      * Years since last BBBM test before this test.
     -
+  * - Test-eligibility
+    - Test type (CSF/PET, BBBM)
+    - Number of simulants who are eligible for both baseline (CSF/PET) and BBBM testing, based on the
+      :ref:`PET/CSF <petcsf_requirements>` and :ref:`BBBM <bbbm_requirements>` eligibilty requirements 
+      listed in step 1 for each section (do not include propensity as a requirement). 
+      Will be used to check that eligible simulants * test rate = test count, for each test type and location.
```

Comment:
For both of these observers, this is a lot of detail. Often we will check this type of thing in an interactive sim to avoid having such specific observers which tend to slow down the sim and be huge files that are hard to work with. Maybe connect with Nathaniel and see if any of this V&V should be done interactively and make makes sense to have observers for.